website/docs/reference/appsmith-framework/console-object.md

```diff
@@ -1,223 +1,73 @@
-        let isTaskIncomplete = false;
-        const taskList = getTaskList.data.record;
-        for (const task of taskList) {
-            if(task.ownerId == Table1.selectedRow.ownerId && task.endDate < Date() && task.status != "Completed") {
-                isTaskIncomplete = true;
-                break;
-            }
-        }
-        if (isTaskIncomplete){
-            sendEmailToNotifyUser.sendEmail();
-            return;
-        } 
-        showAlert("No action is needed");
-}
-```
-
-
-The API generates the correct response when executed standalone, and your app code works as expected. However, the code fails during integration because the API response either is not generated or isn't as expected.
-
-To troubleshoot the error, you would want to log some messages: at the start of the API call, the parameters you are building and passing to the API, the response you get from the API, and the result. Here, the console object comes in handy. You can use different methods such as [`log`](console-object.md#log) to log the start of the method, parameters, and result, [`error`](console-object.md#error) to log the error messages returned by the API, and [`warn`](console-object.md#warn) to log the warnings returned by API.
+<figure>
+  <img src="/img/global-objects-console-logs-tab.png" style= {{width:"700px", height:"auto"}} alt="The Logs tab"/>
+  <figcaption align = "center"><i>Logged messages</i></figcaption>
+</figure>
 
-### Log
-
-The `console.log()` method outputs a message to the logs tab. The message could be a single string value, multiple string values, or JavaScript object.
+In Appsmith, you can use the following methods for logging messages:
 
 :::note
-Console methods **don’t** support **string substitutions**.
+Console methods do not support string substitutions.
 :::
 
-For outputting the entry-level messages, parameter values, and end result, you can add the console.log messages as below:
-
-
-```javascript
-export default {
-    notifyUserIfTaskIsIncomplete: async () => {
-        console.log("Entered method- notifyUserIfTaskIsCompleted");
-        let isTaskIncomplete = false;
-        console.log("Selected Owner Id: " + Table1.selectedRow.ownerId);
-        const taskList = getTaskList.data.record;
-        for (const task of taskList) {
-            if(task.ownerId == Table1.selectedRow.ownerId && task.endDate < Date() && task.status != "Completed") {
-                isTaskIncomplete = true;
-                break;
-            }
-        }
-        if (isTaskIncomplete){
-            sendEmailToNotifyUser.sendEmail();
-            return;
-        } 
-        showAlert("No action is needed");
-        console.log("Exitted method- notifyUserIfTaskIsCompleted");
-    }
-}
-```
-
-
-The method entry, exit, and parameter supplied to the method can be logged and viewed in the [logs tab](../../core-concepts/writing-code/javascript-editor-beta/#logs-tab).
-
+* [log](#consolelog-args)
```

Comment:
delete all these bullet points. We are not following this structure anywhere for methods